practice-app/website/api/event.py

```diff
@@ -76,5 +79,70 @@ def event():
         except exc.SQLAlchemyError as e:
             db.session.rollback()
             return "Try Later", 403
-        
+
         return jsonify(new_event.serialize()), 201
+
+
+# When the id of the event given, corresponding discussion is returned by adding the definition of the sport type in the json format
+# Corresponding event must exist and have a sport type in db.
+@events.route('<event_id>/discussions', methods=['GET', 'POST'])
+@swag_from('doc/discussionForEvent_GET.yml', methods=['GET'])
+@swag_from('doc/discussionForEvent_POST.yml', methods=['POST'])
+def discussionForEvent(event_id):
+
+    if request.method == 'GET':
+
+        if int(event_id) <=-1:
+            return "Wrong path parameters", 401
+        try:
+            eventList = Event.query.all()
+        except exc.NoReferenceError as e:
+            return "Database error", 400
+        eventList = Event.query.all()
```

Comment:
this can be a specified query. e.g. Event.query.filter_by(id=event_id)
then the next part will be easier

Reply:
Yes, but because it works at the moment and the time is limited probably it can remain as it is.